lib/Yam/Agama/Pom/GrubMenuAgamaPage.pm

```diff
@@ -29,6 +29,10 @@ sub boot_from_hd {
     send_key 'ret';
 }
 
+sub boot_first_entry {
```

Comment:
If I would be picky then I would name it "boot_current_selection". Its just the context of first calling "boot_from_hd" and then entering the grub that comes from the hard disk that this "send_key ret" boots the first entry.